docs/_documentations/mdt-vsc-getting-started.md

```diff
@@ -12,18 +12,19 @@ parent: mdt-vsc-overview
 
 # Installing Codewind for VS Code
 <br/>
-Codewind for VS Code enables you to develop your containerized projects from within VS Code.
+
+ Codewind for VS Code enables you to develop your containerized projects from within VS Code. 
```

Comment:
I know we don't want to get hung up on style stuff, but here's what a style guide says about "enable": `Avoid stating that inanimate objects grant abilities to people, as in "the product enables you to...".`

I think the imperative would be better, such as something like, "Install Codewind for VS Code to develop your containerized projects from within VS Code." I think this wording helps make the language more user focused, whereas "This product enables you to..." sounds more product focused.